tests/bluetooth/shell/Makefile

```diff
@@ -13,6 +13,10 @@ QEMU_EXTRA_FLAGS = -serial unix:/tmp/bt-server-bredr
 
 ifeq ($(BOARD),arduino_101)
 CONF_FILE = arduino_101.conf
+else ifeq ($(BOARD),nrf52840_pca10056)
```

Comment:
either you include the nRF51 devkit in these if statements or you call the .conf file nrf52.conf

Reply:
Sure.... will fix this and also test the host controller split builds for throughput